packages/material-ui/src/Checkbox/Checkbox.test.js

```diff
@@ -14,17 +16,21 @@ describe('<Checkbox />', () => {
     mount = createMount({ strict: true });
   });
 
-  after(() => {
-    mount.cleanUp();
-  });
+  describeConformance(<Checkbox />, () => ({
+    mount,
+    only: ['refForwarding'],
+    refInstanceof: window.HTMLSpanElement,
+    after: () => mount.cleanUp(),
+  }));
 
+  /* TODO Checkbox violates root component
```

Comment:
Added as a workaround as per @eps1lon's change [for Switch](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Switch/Switch.test.js#L25)

Reply:
Why is this not working any longer? I wouldn't expect this to affect any tests in our conformance suite.

Reply:
We've switched from `getByTestId` to `getByRole` as per this [comment](https://github.com/mui-org/material-ui/pull/19319#discussion_r369242462).  With the previous `describeConformance` the `getByRole` finds two checkboxes and the rendered HTML doesn't match what we expect (please see [gist](https://gist.github.com/rostislavbobo/653c78112a7c48e11b0bb713bdde4c4e))
If you uncomment your's TODO in `Switch.test.js`, the tests won't pass (both yours and mine)

Reply:
Found the way to fix the tests without workarounds, sorry for the mess